spring-boot-project/spring-boot-docs/src/docs/asciidoc/production-ready-features.adoc

```diff
@@ -1773,15 +1756,16 @@ It is recommended to limit scope to only this one permission.
 	      dynatrace:
 	        api-version: "v2"
 	        api-token: "YOUR_TOKEN"
-	        uri: "YOUR_URI"
+	        uri: "https://{your-environment-id}.live.dynatrace.com/api/v2/metrics/ingest"
+            # or "https://{your-domain}/e/{your-environment-id}/api/v2/metrics/ingest" for managed deployments.
```

Comment:
```suggestion
	        # or "https://{your-domain}/e/{your-environment-id}/api/v2/metrics/ingest" for managed deployments.
```
Otherwise it's not aligned when rendered (tabs vs spaces).